### DISCUSSION TITLE
Add Shapes properties to Container or Stack

### DISCUSSION THREAD

**Discussion by ingdesarr1** [flet-dev/flet#1926] [Ideas]
Good morning, I like to suggest to add the Shapes properties, present in the Canvas element, to the Container and/or the Stack element. This property can facilitate the Responsive design in many cases, such as managing images, texts and other elements. 

Thanks

**Comment by plugeit** [comment]
for that purpose there is container and using border radius you can make it a circle, but there are limitations like polygons

**Reply by ingdesarr1** [reply]
No, I don't want to make a shape like circles, or else. Shapes is a property of Canvas that allows take the shape of width and heigth of the canvas. If this option existed in Container, I can modify or resize images according of their size.
